cgmml/common/evaluation/CV/evaluation.py

```diff
@@ -32,7 +32,7 @@
 METADATA_ANGLE = 14
 
 
-def filter_metadata(indata: list) -> list:
+def filter_metadata(indata: list, standing: bool) -> list:
```

Comment:
```suggestion
def filter_metadata(indata: list, is_standing: bool) -> list:
```

https://www.comp.nus.edu.sg/~cs2103/AY1617S1/contents/coding-standards-java.html